src/foam/i18n/scripts.jrl

```diff
@@ -0,0 +1,243 @@
+p({
```

Comment:
Is this a test script? Asking as I don't understand the the locale, variant, ... settings in the forEach.

Reply:
Yes, it is for test.
But we can change it in the production.